### ISSUE TITLE
Feedback

### ISSUE THREAD

**Issue by felixbuenemann** [jassa/hyper_api#1]
This gem looks very interesting, but I think the name is misleading, given all the hype around Hypermedia APIs, which it got nothing to do with. It also doesn't currently handle links, which are the Hyper part of HTML, but only really parses markup into an object.  Maybe something along the lines of HTML Object Mapper would be a better name (HOM, HTMLMapper).

Featurewise I think it would be useful to integrate the HTTP layer into the gem, so it could handle associations (following mapped links and get new objects of the target pages object type). That way you would be putting the Hyper part in HyperAPI :smile:

**Comment by jassa** [comment]
Yeah, I see your point and I agree, `hyper_api` is only missing the `media` part from it and it becomes an entirely different term. But still `hyper` and `api` are key terms behind the idea and purpose of this gem, which is to help you build an API directly off the HTML of services that don't have one. 

Regarding the integration of the HTTP layer.. I think I'll use the gem for a while before I see what's next, I'd like to keep it simple.

Thanks for your feedback.. I'll keep both things in mind.
